.github/workflows/e2e-semgrep-ci.yml

```diff
@@ -0,0 +1,123 @@
+name: Semgrep CI E2E Test
+on:
+  workflow_dispatch:
+    inputs:
+      docker_tag:
+        description: "Docker Tag to Run. Default: edge"
+        required: false
+        default: "edge"
+  schedule:
+    - cron: "43 20 * * *"
+
+jobs:
+  get-inputs:
+    name: Get Inputs
+    runs-on: ubuntu-22.04
+    outputs:
+      docker_tag: ${{ steps.get-inputs.outputs.docker_tag }}
+    steps:
+      - name: Set variables
+        id: get-inputs
+        env:
+          DEFAULT_TAG: edge
```

Comment:
What is edge vs develop?

Reply:
I think they may be the same - I'm going to switch to `develop` for clarity, though.